controller/value_resolver.rst

```diff
@@ -356,3 +440,34 @@ command to see which argument resolvers are present and in which order they run:
 .. code-block:: terminal
 
     $ php bin/console debug:container debug.argument_resolver.inner --show-arguments
+
+You can also configure the name passed to the ``ValueResolver`` attribute to target
+your resolver. Otherwise it will default to the service's id.
+
+``controller.targeted_value_resolver``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set this tag if you want your resolver to be called only if it is pinned by a
+``ValueResolver`` attribute. Like ``controller.argument_value_resolver``, you
+can customize the name by which your resolver can be targeted.
+
+As an alternative, you can add the
+:class:`Symfony\\Component\\HttpKernel\\Attribute\\AsTargetedValueResolver` attribute
+to your resolver and pass your custom name as its first argument::
+
+    // src/ValueResolver/IdentifierValueResolver.php
+    namespace App\ValueResolver;
+
+    Symfony\Component\HttpKernel\Attribute\AsTargetedValueResolver;
+    use Symfony\Component\HttpKernel\Controller\ValueResolverInterface;
+
+    #[AsTargetedValueResolver('booking_id')]
+    class BookingIdValueResolver implements ValueResolverInterface
```

Comment:
If possible, please let's add a real example below this showing a controller using this new targeted resolver. Thanks

Reply:
How can I split the two code blocks? Adding a line break does not work :thinking:

Reply:
Yes, this is not simple. The best way is to add a short phrase before the second example.

E.g. `This is how you can use this targeted resolver in your controller::`

Reply:
Okay done: https://github.com/symfony/symfony-docs/pull/17763/commits/fa8e76b5b95aa676f72450a8d1fae2f5135f28c4